types/zz_generated_deepcopy.go

```diff
@@ -249,6 +249,21 @@ func (in *BaseService) DeepCopyInto(out *BaseService) {
 			(*out)[key] = val
 		}
 	}
+	if in.ExtraArgsArray != nil {
```

Comment:
`go generate` results should be in a separate commit.

Reply:
fixed